rental.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'book'
+require_relative 'person'
+
+attr_accessor :date, :book, :person
+
+class Rental
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
+
+  def add_rental
+    book.rentals << self
+  end
+
+  def add_to_person
+    person.rentals << self
+  end
+end
```

Comment:
- [ ] The `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental` is not complete yet. Please note that a person should have many rentals. Please add a method to the Person class for adding rentals, and add an instance variable @rentals to the initialize method in the Person class.